Taskfile.yml

```
@@ -324,56 +322,50 @@ tasks:
       - pre-commit run -a
 
   run-website:
-    desc: Build & serve the static website.
+    desc: Build & serve the static website for interactive development.
     dir: web/website
     cmds:
       - hugo server
 
-  run-web:
-    desc: Build & serve the website & playground.
-    summary:
-      Note that this only live-reloads the static website; and requires
-      rerunning to pick up playground & book changes.
-    dir: web/website
-    cmds:
-      - task: build-web
-      # Then start web server with rendering to disk, so it picks up the playground files.
-      - hugo server --renderToDisk
-
   run-book:
-    desc: Build & serve the book.
+    desc: Build & serve the book for interactive development.
     dir: web/book
     cmds:
       - mdbook serve --port=3000
 
   run-playground:
-    desc: Build & serve the playground.
-    dir: web/playground
-    cmds:
-      - npm install
-      - npm start
-
-  run-playground-cached:
-    desc: Build & serve the playground, without rebuilding rust code.
+    desc: Build & serve the playground for interactive development.
     dir: web/playground
     cmds:
-      - task: install-playground-npm-dependencies
+      - task update-playground-dependencies
       - npm start
 
-  install-playground-npm-dependencies:
-    # Look to see if npm dependencies need to be updated
-    # Use task's sources/generates to see if checksums of
-    # node_modules directory have changed since package.json was updated
-
-    desc: Check to see if package.json has changed since we ran npm install
+  update-playground-dependencies:
```

```suggestion
  install-playground-dependencies:
```

...or any particularly reason to deviate?